common/app/views/fragments/labourLiverpoolSeries.scala.html

```diff
@@ -48,9 +48,12 @@
                 }
             }
         </div>
-        <div class="content--labour-liverpool-article__headline-background">
+        <div class="content--labour-liverpool-article__headline-background hide-on-mobile">
              @fragments.inlineSvg("labour-liverpool-headline-bg", "membership")
         </div>
+        <div class="content--labour-liverpool-article__headline-background-mobile mobile-only">
+             @fragments.inlineSvg("labour-liverpool-headline-bg-mobile", "membership")
```

Comment:
Are these really different svgs? Can't we just change their size in css and not load two pictures?